pkg/kubelet/nodeshutdown/storage.go

```diff
@@ -70,7 +72,7 @@ func atomicWrite(filename string, data []byte, perm os.FileMode) error {
 	if err != nil {
 		return err
 	}
-	err = os.Chmod(f.Name(), perm)
+	err = volumeutil.Chmod(f.Name(), perm)
```

Comment:
For the [unit test](https://github.com/kubernetes/kubernetes/blob/127f33f63d118d8d61bebaba2a240c60f71c824a/pkg/kubelet/nodeshutdown/storage_test.go#L57) for `Store` (which calls this), will it be possible to add a unit test for checking for the permissions here?